app/controllers/items_controller.rb

```diff
@@ -77,4 +79,11 @@ def scrape_with_url(raw_url)
   def item_already_exists?
     @item.errors[:item_url][0] == "has already been taken"
   end
+
+  protected
```

Comment:
Change this to `private`... `protected` is valuable for making a method callable by members of the same (or inherited) classes, while `private` is intended to make something callable only within itself...